realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -1618,6 +1630,95 @@ public void buildQueryFromResultsGC() {
         }
     }
 
+    private static byte[][] binaries = {{1, 2, 3}, {1, 2}, {1, 2, 3}, {2, 3}, {2}, {4, 5, 6}};
+
+    private void createBinaryOnlyDataSet() {
+        realm.beginTransaction();
+        realm.delete(BinaryOnly.class);
```

Comment:
This is not needed. `TestRealmConfigurationFactory` will ensure every test starts with a empty Realm.